c10/util/Exception.cpp

```diff
@@ -111,7 +111,8 @@ WarningHandler* get_warning_handler() noexcept(true) {
 void WarningHandler::process(
     const SourceLocation& source_location,
     const std::string& msg) {
-  std::cerr << "Warning: " << msg << " (" << source_location << ")\n";
+  LOG_AT_FILE_LINE(WARNING, source_location.file, source_location.line)
```

Comment:
@dzhulgakov this is problematic because it means that we cannot use `TORCH_WARN` in code that runs during static initialization (before google logging is initialized).
Should that macro ensure that google logging is initialized before calling anything?